komand/server.py

```diff
@@ -4,10 +4,23 @@
 from flask import Flask, jsonify, request, abort
 from gunicorn.arbiter import Arbiter
 from gunicorn.six import iteritems
+from apispec import APISpec
+from apispec.ext.marshmallow import MarshmallowPlugin
+from apispec_webframeworks.flask import FlaskPlugin
+from marshmallow import Schema, fields
 
 from .exceptions import ClientException, ServerException, LoggedException
 
 
+class PluginSchema(Schema):
```

Comment:
Should this be specific to Plugin Info?
```suggestion
class PluginInfo(Schema):
```